windows-installer/fluent-template.conf

```diff
@@ -1,5 +1,7 @@
 # Master configuration template file for google-fluentd on windows.
 
+@include CUSTOM_CONFIG_DIR_PLACE_HOLDER
```

Comment:
Why not just `@include config.d/*.conf`, to make it a directory relative to the location of `fluentd.conf`? You could use a placeholder variable for `config.d`, to ensure that it doesn't fall out of sync with the code in the installer that creates it.